test_runner/fixtures/neon_fixtures.py

```diff
@@ -158,6 +174,21 @@ def neon_binpath(base_dir: Path, build_type: str) -> Iterator[Path]:
     yield binpath
 
 
+@pytest.fixture(scope="function")
+def compatibility_neon_binpath(base_dir: Path, build_type: str) -> Optional[Iterator[Path]]:
+    if os.getenv("REMOTE_ENV"):
+        return
+    if env_compatibility_neon_binpath := os.environ.get("COMPATIBILITY_NEON_BIN"):
+        comp_binpath = Path(env_compatibility_neon_binpath).resolve()
+        if not (comp_binpath / "pageserver").exists():
+            raise Exception(f"compatibility neon binaries not found ad {comp_binpath}")
```

Comment:
I don't think this check for `pageserver` is required here

Reply:
removed